src/components/Calculator.js

```diff
@@ -1,31 +1,42 @@
-import { React } from 'react';
+import { React, useState } from 'react';
 import '../index.css';
+import calculate from '../logic/calculate';
 import Output from './Output';
 import Buttons from './Buttons';
 
 export default function Calculator() {
+  const [obj, setObj] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const addTotext = (val) => {
+    setObj(calculate(obj, val));
+  };
+
   return (
     <div className="calc-grid">
-      <Output data-testid="my-output" />
+      <Output data-testid="my-output" total={obj.total} next={obj.next} />
       <Buttons type="button" symbol="AC" className="clear" />
```

Comment:
- [ ] When clinking on this button `AC`, the app returns this error below

<img width="804" alt="1" src="https://github.com/addisalemseifu/mathapp/assets/77843050/4a5f1271-5478-4132-9b58-116987832255">

- [ ] I think this button should reset the input field, please take a look at calculate.js and fix this issue